src/Runtime/LambdaRuntime.php

```diff
@@ -42,19 +43,33 @@ final class LambdaRuntime
     /** @var Invoker */
     private $invoker;
 
-    public static function fromEnvironmentVariable(): self
+    /** @var int seconds */
+    private $timeout;
+
+    public static function fromEnvironmentVariable(?int $timeout = null): self
     {
-        return new self((string) getenv('AWS_LAMBDA_RUNTIME_API'));
+        return new self((string) getenv('AWS_LAMBDA_RUNTIME_API'), $timeout ?? (int) getenv('BREF_TIMEOUT'));
```

Comment:
If the BREF_TIMEOUT env var does not exist, the `getenv()` method will return `false`. And `(int) false` will be 0.

Reply:
This is covered with tests